config/sync/core.entity_form_display.node.moj_pdf_item.default.yml

```diff
@@ -15,16 +15,62 @@ dependencies:
     - field.field.node.moj_pdf_item.field_moj_thumbnail_image
     - field.field.node.moj_pdf_item.field_moj_top_level_categories
     - field.field.node.moj_pdf_item.field_prison_categories
+    - field.field.node.moj_pdf_item.field_release_date
```

Comment:
why is this in a file called pdf_item?

Reply:
There is a content type called "PDF item" which this file refers to.  These can be part of a series on the Hub (along with audio items, video items, and basic pages). 
We did take a look at removing PDF items from series, but there is already quite a bit of content that is created as pdf's as part of a series.